app/graphql/types/query_type.rb

```diff
@@ -143,4 +143,12 @@ def find_category_by_id(id:)
   def find_category_by_slug(slug:)
     ::Category.find_by(slug: slug)
   end
+
+  field :mapping, Types::Mapping.connection_type, null: false do
+    description 'All Mappings in the Kitsu database'
+  end
+
+  def mapping
+    ::Mapping.all
+  end
```

Comment:
I don't think this is necessary or useful, how about we just expose `Mapping.lookup` like this

```gql
type Query {
  lookupMapping(externalSite: String!, externalId: ID!): MappingItem
}
```

Reply:
Ok, I'll also like to add one for just gettin all based on an `externalSite` which I think can be useful. Will also make that `externalSite` an Enum so the user knows what options they have.

Reply:
`findMapping` to remain consistent with the API terms

Reply:
> `findMapping` to remain consistent with the API terms

I was thinking that but it isn't really finding a mapping, it is finding the mapping item.

Reply:
`findMapping` to remain consistent with the rest of the GraphQL API?